DATA/SConscript

```diff
@@ -174,16 +174,22 @@ env.Alias("crosstalk", crosstalkGen)
 # Defects
 defectExposure = "^".join([str(vv) for vv in
                            exposureDict['flatExposures'] + exposureDict['darkExposures']])
-defectsGen = env.Command(os.path.join(REPO_ROOT, 'defectGen'), flat,
-                         [getExecutableCmd('cp_pipe', 'findDefects.py', REPO_ROOT,
-                                           f"--calib {CALIB_ROOT}", "--rerun",
-                                           f"{REPO_ROOT}/defectGen",
-                                           "--id detector=0", f"--visitList {defectExposure}",
-                                           "-c assertSameRun=False",
-                                           "-c isrForDarks.doFlat=False",
-                                           "-c isrForFlats.doFlat=False",
-                                           "-c writeAs=BOTH")])
-env.Alias('defectsGen', defectsGen)
+defectIsr = env.Command(os.path.join(REPO_ROOT, "defectIsr"), flat,
```

Comment:
Is this doing ISR on the master flat and master dark?  Or is this flattening both the master dark and the master flat?

Reply:
This pre-processes the inputs through ISR.  Both `doDark` and `doFlat` are set to `False` in the `cp_pipe/config/defectIsr.py` that contains the config file for this.  This could likely depend only on the `bias` being generated instead of the `flat`.  I think I made that the dependency for everything that isn't the `bias` or `dark` to avoid needing to choose more specific dependencies.